docs/developers/contributing/performance/index.md

```diff
@@ -0,0 +1,7 @@
+(performance)=
+# Performance
+
+Information on profiling and benchmarking.
+
+```{tableofcontents}
```

Comment:
This is just showing a blank area
<img width="1564" alt="image" src="https://github.com/napari/docs/assets/76622105/275571e0-633a-4d25-bc61-955709aa4923">

But this could be due to the issue in the _toc.yml